### DISCUSSION TITLE
maven dependency not found

### DISCUSSION THREAD

**Discussion by t0nikro0s** [soot-oss/SootUp#737] [Q&A]
While using sootup.java.bytecode v1.1.2 in maven, it need com.github.ThexXTURBOXx.dex2jar dependency.
But I can not find it in any maven repository
how can I deal with this problem

**Comment by jpstotz** [comment]
`com.github.ThexXTURBOXx.dex2jar` is a dependency that is provided by [Jitpack.io](https://jitpack.io) repository.

Jitpack.io dynamically provides GitHub repos as Maven/Gradle/... artifact.
It is defined as repository in SootUp here: https://github.com/soot-oss/SootUp/blob/647da14028d93f58a9b416160c99a7569cf3bce8/sootup.java.bytecode/pom.xml#L44-L47

**Reply by t0nikro0s** [reply]
thanks for helping first.
I tried to use jitpac.io as repository, but it seems that something failed while transfering.
<img width="796" alt="4c9051226028a45458f9e798355c48e" src="https://github.com/soot-oss/SootUp/assets/92668186/331585bc-d0fe-41f3-ac8e-c64f158fca71">

**Comment by jpstotz** [comment]
`https://jitpack.io/com/github/ThexXTURBOXx/dex2jar/dex-tools/v64/dex-tools-v64.jar` and it's pom can be downloaded without problems. Most likely this was a temporary problem on network or server level. As Maven caches download errors it will not retry downloading the artifact for the next days. 

My recommendation is to delete the directory `~/.m2/repository/com/github/ThexXTURBOXx/dex2jar` to detel the cached error and retry.

**Comment by Liyw979** [comment]
Or you can just exclude dex2jar , I am doing this too.
`https://www.baeldung.com/mvn-plugin-dependency-exclusion`

**Reply by t0nikro0s** [reply]
Thanks, It helps!
